ppr-ui/src/components/parties/party/SecuredParties.vue

```diff
@@ -541,7 +541,7 @@ export default defineComponent({
 
     const fetchOtherSecuredParties = async () => {
       localState.loading = true
-      if (parties.registeringParty) {
+      if (localState.loading) {
```

Comment:
this seems wrong haha? loading is set to true right above this statement? Do you mean to take out this 'if' entirely?

Reply:
well, in a recent change, registering Party is set to null, so that no longer works, but it's not really needed anyways. If you look at the function, it tries to load the secured parties and if successful, it sets loading to false, otherwise sets a timeout for 3 seconds and calls the function again.

Reply:
Sorry for the late response! Okay sounds good, can you take out the 'if' then? I can see that confusing us later haha

Reply:
removed the if